config/__init__.py

```diff
@@ -299,11 +302,13 @@
 TRADING_MODE_REQUIRED_STRATEGIES_MIN_COUNT = "required_strategies_min_count"
 STRATEGIES_REQUIRED_EVALUATORS = "required_evaluators"
 TENTACLE_DEFAULT_CONFIG = "default_config"
+TENTACLE_DEFAULT_FOLDER = "Default"
 
 # Web interface
 UPDATED_CONFIG_SEPARATOR = "_"
 GLOBAL_CONFIG_KEY = "global_config"
 EVALUATOR_CONFIG_KEY = "evaluator_config"
+DEACTIVATE_OTHERS = "deactivate_others"
 TRADING_CONFIG_KEY = "trading_config"
 COIN_MARKET_CAP_CURRENCIES_LIST_URL = "https://api.coinmarketcap.com/v2/listings/"
```

Comment:
In an other PR we should implement internationalization :earth_africa: :earth_americas:  :earth_asia: